### PR TITLE
support for SGOS

### PR DESCRIPTION
**Pull request by seekerOK**
Add sgos.rb to models and to README

### CODE REVIEW

File: lib/oxidized/model/sgos.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    cfg.gsub! /^(download encrypted-password).*/, ""
+    cfg
+  end
+
+  cfg :telnet, :ssh do
+    # preferred way to handle additional passwords
+    if vars :enable
+      post_login do
+        send "enable\n"
+        cmd vars(:enable)
```
Comment (danilopopeye) [inline lib/oxidized/model/sgos.rb]: @seekerOK just curiosity but why the mix of `#send` and `#cmd` here?

**Comment by danilopopeye** [comment]
@seekerOK did you managed to find out which chars are those in [your comment](https://github.com/ytti/oxidized/issues/997#issuecomment-326009264) ?

**Comment by seekerOK** [comment]
this is what i get in the debug log **Ex. ^H**

```
Licensable component information:

Component name:         SGOS 6 SWG Edition
Valid:                  yes
Serial number:          111111
Product Description:    Secure Web Gateway Virtual Appliance Model 100 with 250 users, 3 yr subscription
Part Number:           1111111111
Activation date:        2016-01-07
Expiration date:        2018-12-30

--More--^H^H^H^H^H^H^H^H        ^H^H^H^H^H^H^H^HComponent name:         Windows Media Streaming
```

And in the saved config:

```
!- Licensable component information:
!- 
!- Component name:         SGOS 6 SWG Edition
!- Valid:                  yes
!- Serial number:          1111111111
!- Product Description:    Secure Web Gateway Virtual Appliance Model 100 with 250 users, 3 yr subscription
!- Part Number:            1111111111
!- Activation date:        2016-01-07
!- Expiration date:        2018-12-30
!- 
!- ��������        ��������Component name:         Windows Media Streaming
```


And i use this for a BlueCoat\Symantec ProxySG V100

**Comment by danilopopeye** [comment]
@seekerOK can you try two variations of some code for me?

``` ruby
cmd 'show general' do |cfg|
  comment cfg.delete("\b")
end
```

and


``` ruby
cmd 'show general' do |cfg|
  comment cfg.gsub(/\^H/, '')
end
```

one of these should remove those `^H` from the backup 🤓

**Comment by seekerOK** [comment]
same results 😞

**Pull request merged**